### DISCUSSION TITLE
缺少文件 'main.bbl' 编译通过不过，该如何处理？

### DISCUSSION THREAD

**Discussion by erwincv** [sjtug/SJTUThesis#1054] [提问]
在TeXStudio中编译latex出现这样的错误。
![image](https://github.com/user-attachments/assets/4e80b009-054d-4e57-a4e4-e37555179b9f)
日志如下：
![image](https://github.com/user-attachments/assets/bdedbcd1-fc13-447b-8c26-62c939cf2d5b)
 请大佬指点，如何通过编译？谢谢。

**Comment by LogCreative** [comment]
删除临时文件后再重新编译试试？

**Reply by erwincv** [reply]
试过多次，重复出现这个问题。 -_-|||

**Reply by LogCreative** [reply]
最好提供更多的日志信息。

**Reply by erwincv** [reply]
我一点一点，把新版本内容往旧版本上搬，最后walk around这个问题，虽然没有找到root cause，也不打紧了。谢谢。
